labcore/analysis/plotting/holo.py

```diff
@@ -410,22 +420,43 @@ def __init__(self, *args: Any, **kwargs: Any):
         self.pre_process_dim_input = pn.widgets.TextInput(
             value="repetition", name="Pre-process dimension"
         )
+        self.file_loc = pn.widgets.TextInput(
+            name="File Location", value=""
```

Comment:
Does this means that if someone else adds a default value for a child implementation of a LoaderNode the text will still be `""` in the TextInput?